packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -214,22 +214,31 @@ export default class Meetings extends WebexPlugin {
           }
         })
           .finally(() => {
+            // There will be cases where locus event comes in gets created and deleted because its a 1:1 and meeting gets deleted
+            // because the other user left so before sending 'added' event make sure it exists in the collection
+
+            if (this.getMeetingByType('id', meeting.id)) {
             // Had to put `meeting:added here as the parsing of the locus object is done here
-            Metrics.postEvent({event: eventType.NOTIFICATION_RECEIVED, meeting, data: {trigger: trigger.MERCURY_EVENT}});
-
-            Metrics.postEvent({event: eventType.REMOTE_STARTED, meeting, data: {trigger: trigger.MERCURY_EVENT}});
-            Trigger.trigger(
-              this,
-              {
-                file: 'meetings',
-                function: 'handleLocusEvent'
-              },
-              EVENT_TRIGGERS.MEETING_ADDED,
-              {
-                meeting,
-                type: meeting.type === _MEETING_ ? _JOIN_ : _INCOMING_
-              }
-            );
+              Metrics.postEvent({event: eventType.NOTIFICATION_RECEIVED, meeting, data: {trigger: trigger.MERCURY_EVENT}});
```

Comment:
Do we need to post this metrics event even if the meeting isn't in the collection?

Reply:
No the metrics only need to be send when the meeting object exists . We added that check because sometimes more then one mercury event comes in at the same time